From: Cole Robinson <crobinso@redhat.com>
Date: Tue, 30 May 2017 18:35:04 -0400
Subject: [PATCH] daemon: Don't run if in a Fedora live VM

Only start libvirtd if not in a VM OR if not in a Fedora live env,
via systemd unit conditions. It checks Fedora live env by looking
for the rd.live.image option on the kernel command line.

Roundabout way to prevent the 'default' network killing connectivity
of the livecd running in a VM

https://bugzilla.redhat.com/show_bug.cgi?id=1146232

Not upstream, will likely need some kind of different permanent solution
---
 daemon/libvirtd.service.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/daemon/libvirtd.service.in b/daemon/libvirtd.service.in
index fbaf02f3b..86ee988fa 100644
--- a/daemon/libvirtd.service.in
+++ b/daemon/libvirtd.service.in
@@ -16,6 +16,12 @@ After=local-fs.target
 After=remote-fs.target
 Documentation=man:libvirtd(8)
 Documentation=http://libvirt.org
+# This says, start libvirtd if not in a VM OR if not in a Fedora live env
+# Roundabout way to prevent the 'default' network killing connectivity
+# of the livecd running in a VM
+# https://bugzilla.redhat.com/show_bug.cgi?id=1146232
+ConditionVirtualization=|0
+ConditionKernelCommandLine=|!rd.live.image
 
 [Service]
 Type=notify
